build/Linux+BSD/musescore.xml.in

```diff
@@ -108,6 +108,14 @@
         <icon name="application-vnd.recordare.musicxml@MSCORE_INSTALL_SUFFIX@"/>
     </mime-type>
 
+    <mime-type type="application/vnd.recordare.musicxml">
+        <!-- See "Container file" at http://www.musicxml.com/for-developers/musicxml-dtd/ -->
+        <comment>MusicXML compressed score</comment>
+        <glob pattern="*.mxml"/>
```

Comment:
I've no idea about this kind of file, but should/could the mxml type be added as a second `glob` in the mime-type above? I.e.
```xml
    <!-- MusicXML documents -->

    <mime-type type="application/vnd.recordare.musicxml">
        <!-- See "Container file" at http://www.musicxml.com/for-developers/musicxml-dtd/ -->
        <comment>MusicXML compressed score</comment>
        <glob pattern="*.mxl"/>
        <glob pattern="*.mxml"/>
        <sub-class-of type="application/zip"/>
        <icon name="application-vnd.recordare.musicxml@MSCORE_INSTALL_SUFFIX@"/>
    </mime-type>

    <mime-type type="application/vnd.recordare.musicxml+xml">
...
```

Reply:
Probably

Reply:
It sure is, as it is done for .musicxml and .xml already ;-)

Reply:
Done